packages/protocol/thread/thread.ts

```diff
@@ -600,6 +601,38 @@ class _ThreadFront {
     return rv;
   }
 
+  // Same as evaluate, but returns the result without wrapping a ValueFront around them.
+  // TODO Replace usages of evaluate with this.
+  async evaluateNew({
+    asyncIndex,
+    text,
+    frameId,
+    pure = false,
+  }: {
+    asyncIndex?: number;
+    text: string;
+    frameId?: FrameId;
+    pure?: boolean;
+  }) {
+    const pause = await this.pauseForAsyncIndex(asyncIndex);
+    assert(pause, "no pause for asyncIndex");
+
+    const rv = await pause.evaluate(frameId, text, pure);
+
+    if (repaintAfterEvaluationsExperimentalFlag) {
+      const { repaint } = await import("protocol/graphics");
+      repaint(true);
```

Comment:
I wonder if this is something we should do? The method I copied this from _didn't_ but maybe that was a problem?